packages/client/src/declarations/index.ts

```diff
@@ -720,7 +700,11 @@ export async function getDeclarationsOfCurrentUser(): Promise<string> {
   let currentUserDeclarations: IDeclaration[] =
     (currentUserData && currentUserData.declarations) || []
 
-  if (SystemRoleType.FieldAgent.includes(currentUserRole) && currentUserData) {
+  if (
+    '@TODO: Ask what we are going to do with the field agent downloaded records getting pruned in 24 hours' ===
+      ('no' as any) &&
+    currentUserData
+  ) {
```

Comment:
@jpye-finch Field agent downloads are getting removed in 24 hours. This is a special functionality for FA's. How would we handle this in the future?

Reply:
This will also be removed when we refactor and remove 
https://github.com/opencrvs/opencrvs-core/pull/7301#discussion_r1670783517

Reply:
it exist because currently a FA can assign and download records to see the history